### PR TITLE
Cloud sites image pruning 160622 1

### PR DESCRIPTION
**Pull request by kylelaffoon**
- Relates to #705 
Adjusting images and verbiage in the following articles:
 - Troubleshooting your PHP script
 - .NET 4.5 in Cloud Sites
 - Optimize SugarCRM on Cloud Sites
 - Purge Cloud Sites Webmail
 - Rackspace Cloud Sites Essentials: Cloud Sites Technologies

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f8dd642c3f/how-to/)

File: content/cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies.md [hunk 1]
```diff
@@ -47,21 +47,14 @@ to change this technology.
 1.  After you've logged into the Cloud Sites Control Panel, navigate to
 **Hosting > Cloud Sites**.
 
-  <img src="{% asset_path cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies/hosting.png %}" alt="" />
+2.  Click the domain name to which you would like to make technology changes.
 
-2.  Click on the domain name to which you would like to make technology changes.
-3.  Click on the **Features** tab.
+3.  Click he **Features** tab.
```
Comment (catlook) [inline content/cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies.md]: should be Click the **Features** tab.

File: content/cloud-sites/net-45-in-cloud-sites.md [hunk 2]
```diff
@@ -25,36 +25,32 @@ technology option.
 If your websites are currently .NET 4.5-compatible we recommend that you
 perform the .NET 4.5 upgrade. Customers can upgrade to .NET 4.5 by
 selecting the .NET 4.5 technology option from the Features page of the
-control panel. This migration is a very easy process. Please follow
+control panel. This migration is a very easy process. Follow
 these simple steps:
```
Comment (catlook) [inline content/cloud-sites/net-45-in-cloud-sites.md]: I think Kelly would say don't use simple here.  just cut it.

File: content/cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies.md [hunk 3]
```diff
@@ -79,23 +72,19 @@ each of your domains.
...
 
 3.  Scroll down to **Website Features**.
 
-  <img src="{% asset_path cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies/web_features.png %}" alt="" />
-
-What you have here is a breakdown of your website's current features
-including its currently enabled technologies, its number of
+Displayed under **Website Features** is a breakdown of your website's current
+features including its currently enabled technologies, its number of
 databases and whether the Raw Logs are enabled or disabled. (Raw Logs
```
Comment (catlook) [inline content/cloud-sites/rackspace-cloud-sites-essentials-cloud-sites-technologies.md]: add the oxford comma?    databases, and whether ...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-f07f245712/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-16c8c9a674/how-to/). :bow:

**Comment by catlook** [comment]
Golden!!

**Pull request merged**